packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/BasicInformationStep/BasicInformationForm/constants.ts

```diff
@@ -10,6 +10,20 @@ export const existingCommunityNames = app.config.chains
   .getAll()
   .map((community) => community.name.toLowerCase().trim());
 
+const removeTestCosmosNodes = (nodeInfo: NodeInfo): boolean => {
+  return !(
+    window.location.hostname.includes('commonwealth.im') &&
```

Comment:
We are using the magic word 'commonwealth.im' in many places. We might eventually need a DOMAIN constant and some utility functions to check if we are in production domains.